### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by PBWiseman**
I have updated the ReadMe. The features section still needs to be done but I think that might be better done once we have more of the features we want to show off actually in the game? We also need a cover image which can also be added later. Finally we should have a credits section for any assets used. Sounds and music maybe?

I also fixed the title of the pull request template as I noticed that's not working at the moment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,67 @@
...
-We only currently have the game working for 64-bit Windows operating systems.
+| Action | Control |
+|-|-|
+| Movement | WASD | 
+| Shooting | Left click |
+| Aiming | Mouse movement | 
+
+## Get Involved
+
+Join the Snowball Slam community and stay up to date with the latest developments, bug fixes, and upcoming features:
```
Comment (Loerad) [inline README.md:39]: Game is Duck (working title) not snowball slam

File: README.md [hunk 2]
```diff
@@ -1,22 +1,67 @@
...
-We are a student development team and we aren't working on this game 24/7, so development will be slow but we are commited to making this game better over time. 
 
-### Current Alpha Development Map
+<!-- ## Features 
+Need a features tab. Possibly talking about the game loop and the different weapons and upgrades?
+-->
 
-27th of May: Map and Terrian Update
+## Team
+We are the Studio 5+6 Games Team! We are a team in our third year at the Bachelor of Information Technology in Otago Polytechnic. The game started development at the start of 2024 and has been in active development since then. In July, the team experienced a shuffle, with two developers moving to another project and one new developer joining.
```
Comment (Loerad) [inline README.md:13]: Development started in late 2023

Comment (PBWiseman) [reply README.md]: Changed

File: README.md (hunk 1 above)
Comment (PBWiseman) [reply README.md]: Dammit thought I changed all of them. Fixed.

**Review by AlexReidNZ — APPROVED** [review]
The new readme is great, good to be merged

**Review by kblack89 — APPROVED** [review]
Excellent readme, ready for merge.

**Pull request merged**